enterprise/server/hostedrunner/hostedrunner.go

```diff
@@ -197,6 +188,13 @@ func (r *runnerService) createAction(ctx context.Context, req *rnpb.RunRequest,
 		})
 	}
 
+	for k, v := range req.GetEnv() {
```

Comment:
When generating fallback snapshot keys, we only look at cmd.EnvironmentVariables, and not at the --env_overrides flag

Reply:
Makes sense - do we need `env_overrides` anymore after this change or can we remove it?

Reply:
No we don't. Cleaned up in workflow service too. Don't know why I didn't implement this way to begin with...